src/check/parse.zig

```diff
@@ -13,23 +13,23 @@ const exitOnOom = @import("../collections/utils.zig").exitOnOom;
 
 /// Parses a single Roc file.  The returned AST should be deallocated by calling deinit
 /// after its data is used to create the next IR, or at the end of any test.
-pub fn parse(env: *base.ModuleEnv, allocator: std.mem.Allocator, source: []const u8) IR {
+pub fn parse(env: *base.ModuleEnv, source: []const u8) IR {
     var messages: [128]tokenize.Diagnostic = undefined;
     const msg_slice = messages[0..];
-    var tokenizer = tokenize.Tokenizer.init(env, source, msg_slice, allocator);
+    var tokenizer = tokenize.Tokenizer.init(env, source, msg_slice);
     tokenizer.tokenize();
-    const result = tokenizer.finish_and_deinit();
+    const result = tokenizer.finishAndDeinit();
```

Comment:
Does zig have a lint for this? Just curious how you caught the name being in snake_case?

Reply:
I'm the guy that did the stdlib conversion from camelCase to snake_case, I have eyes like an eagle.

(it was just muscle memory from that, not sure if there's an auto-check)